addons/html_builder/static/src/builder.js

```diff
@@ -124,6 +126,9 @@ export class Builder extends Component {
                         }
                         this.removeLoadingEffect();
                     },
+                    on_snippet_dropped_handlers: () => {
+                        this.selectedSnippetEl = null;
+                    },
```

Comment:
Since we discussed it, you can set it to the newly dropped snippet here if you want (just use `{snippetEl}` in the args), now that the entry point is easier :slightly_smiling_face: But as you prefer of course :slightly_smiling_face: